help/en/html/tools/logixng/reference/chapter8.shtml

```diff
@@ -62,14 +64,197 @@
       menu is used to <strong>Delete</strong> a local variable and has <strong>Move Up</strong> and
       <strong>Move Down</strong> options if there is more than one variable.</p>
 
+      <p>Click on <strong>OK</strong> to finish. The local variable will be added to the tree. See
+      the editor image below.</p>
+
+      <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+      <h3>Local variable types</h3>
+
       <div style="margin-left: 2em;">
         <a href="images/chapter8/local_variable_types.png"><img src=
-        "images/chapter8/local_variable_types.png" alt="Chapter 8 local variable types" width="458"
-        height="277"></a>
+        "images/chapter8/local_variable_types.png" alt="Chapter 8 local variable types" width="135"
+        height="188"></a>
       </div>
 
-      <p>Click on <strong>OK</strong> to finish. The local variable will be added to the tree. See
-      the image below.</p>
+      <dl>
+        <dt>None</dt>
+        <dd>Generic type which will change based on its content</dd>
```

Comment:
Any local variable can have any type of data. For example, you can declare the local variable `index` to the integer number 2 and then assign it the string `Hello World`.

The meaning of `None` is that it is initialized to null. None comes from Jython. If this option wasn't here, it wouldn't be possible to tell that a local variable should have the value null.

Reply:
The description for `None` was changed.  Also added a note that the types can change.